onnxscript/test/converter_test.py

```diff
@@ -556,6 +556,14 @@ def test_graph_attr_loop(self):
         expected_output = np.array([0, 1, 3, 6, 10, 15], dtype=np.int64)
         self.check_run(sum_to, inputs, expected_output)
 
+    def test_graph_attr_loop_error(self):
+        from onnxscript.test.models.graph_attr import sum_to_error
+
+        input = np.array(6, dtype=np.int64)
+        with self.assertRaises(ValueError) as exc:
```

Comment:
Consider assertRaisesRegex if trying to match messages? https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex